<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="2">
       <widget class="QScrollArea" name="scrollArea">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAsNeeded</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAsNeeded</enum>
        </property>
        <property name="widgetResizable">
         <bool>false</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>1537</width>
           <height>1337</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <property name="margin">
           <number>0</number>
          </property>
          <property name="spacing">
           <number>0</number>
          </property>
         </layout>
        </widget>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="videoTitle">
        <property name="text">
         <string>Current Frame</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QSlider" name="framePicker">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::NoTicks</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="frameNumber">
          <property name="text">
           <string>0/0</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="operationsTitle">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Operations Pipeline</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" rowspan="2" colspan="2">
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item>
         <widget class="QListWidget" name="listWidget">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QStackedWidget" name="stackedWidget">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>350</width>
            <height>16777215</height>
           </size>
          </property>
          <widget class="QWidget" name="blank">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave_config"/>
    <addaction name="actionLoad_config"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="zoomInAct"/>
    <addaction name="zoomOutAct"/>
    <addaction name="normalSizeAct"/>
    <addaction name="fitToWindowAct"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelp"/>
   </widget>
   <widget class="QMenu" name="menuActions">
    <property name="title">
     <string>Actions</string>
    </property>
    <addaction name="actionApply"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuActions"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionOpen_fluorescence"/>
   <addaction name="separator"/>
   <addaction name="zoomInAct"/>
   <addaction name="zoomOutAct"/>
   <addaction name="normalSizeAct"/>
   <addaction name="fitToWindowAct"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open brightfield</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="zoomInAct">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Zoom In</string>
   </property>
   <property name="shortcut">
    <string>Ctrl++</string>
   </property>
  </action>
  <action name="zoomOutAct">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Zoom Out</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="normalSizeAct">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>100%</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="fitToWindowAct">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Fit to window</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="toolTip">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionApply">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Apply to sequence</string>
   </property>
  </action>
  <action name="actionOpen_fluorescence">
   <property name="text">
    <string>Open fluorescence</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+K</string>
   </property>
  </action>
  <action name="actionSave_config">
   <property name="text">
    <string>Save config</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionLoad_config">
   <property name="text">
    <string>Load config</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actionOpen</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>openImage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>414</x>
     <y>268</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fitToWindowAct</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>fitToWindow()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>414</x>
     <y>268</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>zoomInAct</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>zoomIn()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>414</x>
     <y>268</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>zoomOutAct</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>zoomOut()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>414</x>
     <y>268</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>normalSizeAct</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>normalSize()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>414</x>
     <y>268</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>414</x>
     <y>268</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionApply</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>displayExecutionDialog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>414</x>
     <y>268</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionOpen_fluorescence</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>openImageFluorescent()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>414</x>
     <y>268</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSave_config</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>saveParameters()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>499</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionLoad_config</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>loadParameters()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>499</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>openImage()</slot>
  <slot>zoomIn()</slot>
  <slot>zoomOut()</slot>
  <slot>fitToWindow()</slot>
  <slot>normalSize()</slot>
  <slot>displayExecutionDialog()</slot>
  <slot>openImageFluorescent()</slot>
  <slot>loadParameters()</slot>
  <slot>saveParameters()</slot>
 </slots>
</ui>
